set.mm

```diff
@@ -264005,11 +264112,37 @@ the same dimension over the same (nonzero) ring.  (Contributed by AV,
       CTEUFUGUHUI $.
 
     $( A finite bag is a function.  (Contributed by Mario Carneiro,
-       29-Dec-2014.) $)
-    psrbagf $p |- ( ( I e. V /\ F e. D ) -> F : I --> NN0 ) $=
+       29-Dec-2014.)  Remove extra antecedent.  (Revised by SN,
```

Comment:
It could be "Remove a sethood antecedent.", to give a bit more information. Since sethood antecedents are quite common in set.mm, this gives a good idea of the nature of the revision.  (The fact that it was extraneous is contained in the fact that you could remove it.)